### PR TITLE
158 improvement comments on all server functions

### LINKED ISSUE #158: [IMPROVEMENT]: Comments on all server functions
### Description

Comments on all server functions in app.js and all functions in server/service

### To do

Comments on all server functions

### Priority

Medium

### Project parts

- [X] Server
- [ ] Application
- [ ] Web
- [ ] Automation

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by STom6**
Description
Comments on all server functions in app.js and all functions in server/service

To do
Comments on all server functions

### CODE REVIEW

File: Server/Services/googleService.js [hunk 1]
```diff
@@ -96,8 +99,11 @@ module.exports = {
 }
 
 /**
- * UNDOCUMENTED
- */
+ * function called which returns the content of the message in html format.
+ * @function get_html_message
+ * @param {string} mailContent mailContent is the content of the mail sent.
+ * @returns {*} return string message in htlm format.
```
Comment (JohanCDev) [inline Server/Services/googleService.js]: html

**Review by JohanCDev — CHANGES_REQUESTED** [review]

File: Server/app.js [hunk 2]
```diff
@@ -1,36 +1,150 @@
...
+ * @constant TWITCH_CLIENT_ID
+ * @requires TWITCH_CLIENT_ID
+*/
 const TWITCH_CLIENT_ID = '1ikfbd316i8dggr27rtl8t9x3qrhvf';
+
+/**
+ * @constant TWITCH_SECRET
+ * @requires TWITCH_SECRET
+*/
 const TWITCH_SECRET    = 'je4r7zf1rwv6jn3q8g0fis4lxrgvc0';
```
Comment (31Nathan) [inline Server/app.js:124]: Does not belong here

File: Server/app.js [hunk 3]
```diff
@@ -57,29 +178,67 @@ passport.use('twitch', new OAuth2Strategy({
...
+/**
+ * initialisation of cors in app.
+*/
 app.use(cors());
+/**
+ * initialisation of express.urlencoded() in app. 
+*/
 app.use(express.urlencoded())
+
 //temporaire
```
Comment (31Nathan) [inline Server/app.js:191]: ??

**Review by 31Nathan — CHANGES_REQUESTED** [review]

**Review by CedricCORGE — APPROVED** [review]
ntr

**Review by 31Nathan — APPROVED** [review]
LGTM

**Review by JohanCDev — APPROVED** [review]
lgtm

**Pull request merged**